src/tests/test_package_loading.py

```diff
@@ -270,8 +270,10 @@ def test_test_unvendoring(selenium_standalone):
 def test_install_archive(selenium):
     build_dir = Path(__file__).parents[2] / "build"
     test_dir = Path(__file__).parent
+    # TODO: first arguement actually works as a path due to implementation,
+    # maybe it can be proposed to typeshed?
     shutil.make_archive(
-        test_dir / "test_pkg", "gztar", root_dir=test_dir, base_dir="test_pkg"
+        str(test_dir / "test_pkg"), "gztar", root_dir=test_dir, base_dir="test_pkg"
```

Comment:
Worth checking with typeshed, as this is technically allowed due to the impl, but is not indicated in the Python docs.